src/Sylius/Component/Grid/Definition/ArrayToDefinitionConverter.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Component\Grid\Definition;
+
+/**
+ * @author Paweł Jędrzejewski <pawel@sylius.org>
+ */
+class ArrayToDefinitionConverter implements ArrayToDefinitionConverterInterface
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function convert($code, array $configuration)
+    {
+        $grid =  Grid::fromCodeAndDriverConfiguration($code, $configuration['driver']['name'], $configuration['driver']['options']);
+
+        if (array_key_exists('sorting', $configuration)) {
+            $grid->setSorting($configuration['sorting']);
+        }
+
+        foreach ($configuration['fields'] as $name => $fieldConfiguration) {
+            $field = Field::fromNameAndType($name, $fieldConfiguration['type']);
+
+            if (array_key_exists('path', $fieldConfiguration)) {
+                $field->setPath($fieldConfiguration['path']);
+            }
+            if (array_key_exists('label', $fieldConfiguration)) {
+                $field->setLabel($fieldConfiguration['label']);
+            }
+            if (array_key_exists('options', $fieldConfiguration)) {
```

Comment:
I will take care of this class in the next iteration, probably with a hydrator, yeah. Fortunately it can be refactoring of this one class in a separate PR. No reason to delay this anymore. :)